operator/Makefile

```diff
@@ -106,7 +106,7 @@ help: ## Display this help.
 
 .PHONY: deps
 deps: go.mod go.sum
-	go mod tidy -compat=1.17
```

Comment:
Have you tested that the operator-sdk command still work after removing the compat flag, e.g.
```
operator-sdk create api --group loki --version v1 --kind TestLokiStack --namespaced=true --controller=false --resource --verbose
```

Reply:
as far as I can tell, it works fine (I'm no expert in operator-sdk tho)